h_periodic/lms_beat.py

```diff
@@ -30,7 +30,7 @@
         "rotate-rsa-keys": {
             "options": {"expires": 600},
             "task": "lms.tasks.rsa_key.rotate_keys",
-            "schedule": timedelta(days=1),
+            "schedule": timedelta(hours=1),
```

Comment:
You could schedule this at a specific time of the day rather than at a delay of one day. That way as long as the service isn't down that exact instant, it will still run daily.